src/CashierServiceProvider.php

```diff
@@ -3,6 +3,7 @@
 namespace Laravel\Cashier;
 
 use Illuminate\Support\ServiceProvider;
+use Laravel\Cashier\Cashier;
```

Comment:
No need

Reply:
Yes, I left them there for this exact request to come up. 😂 I'll remove them.

Reply:
Or do you actually want explicit imports in the same namespace removed? Not only obsolete ones?

Reply:
Import statements are removed and submitted. Documentation has been added in a separate section.